scripts/lib/CIME/build.py

```diff
@@ -258,6 +263,9 @@ def _build_libraries(case, exeroot, sharedpath, caseroot, cimeroot, libroot, lid
             full_lib_path = os.path.join(sharedlibroot, sharedpath)
         elif lib == "mpi-serial":
             full_lib_path = os.path.join(sharedlibroot, sharedpath, "mct", lib)
+        elif lib == "cprnc":
+            full_lib_path = os.path.join(sharedlibroot, sharedpath, lib)
```

Comment:
I suggest that here full_lib_path = sharedlibroot.  This will avoid building more copies of cprnc than you need.

Reply:
Good idea. I implemented your suggestion to build cprnc higher up.